src/pages/index.js

```diff
@@ -166,7 +184,15 @@ const IndexPage = ({ data }) => {
                 </div>
               </div>
               <div className="video-wrapper">
-                <img src={tidbFeaturesGIF.publicURL} alt="TiDB-Feature" />
+                {browser.isMiBrowser || browser.isX5Browser ? (
+                  // if browser is MiBrowser or WeChat X5 Browser, show GIF for resolving z-index error
+                  <img src={tidbFeaturesGIF.publicURL} alt="" />
+                ) : (
+                  <video loop muted autoPlay poster={tidbFeaturesPNG.publicURL}>
+                    <source src={tidbFeaturesMP4.publicURL} type="video/mp4" />
+                    <img src={tidbFeaturesGIF.publicURL} alt="" />
```

Comment:
what is the img used for?

Reply:
if visitor's browser doesn't support the video format of MP4, show the GIF image instead of video